mad_prefect/data_assets/data_asset.py

```diff
@@ -219,10 +237,21 @@ async def __call__(self, *args, **kwargs):
             f"Completed operations for asset_run_id: {self.asset_run.id}, on asset_id: {self.id}, on asset: {self.name}"
         )
 
-        return result_artifact
+        return self.result_artifacts[0]
+
+    def _create_result_artifacts(self) -> List[DataArtifact]:
+        base_path = self.path.split(".")[0]
```

Comment:
This may introduce issues when the path is something like bronze/file.test.parquet. Not too worried about this for now, but potentially a bug in the future.